.github/workflows/release_branches.yml

```diff
@@ -236,7 +243,7 @@ jobs:
       - name: Check out repository code
         uses: ClickHouse/checkout@v1
         with:
-          clear-repository: true
+          filter: tree:0
```

Comment:
I think we can remove the whole `with` here, filter is meaningless with default `depth: 0`